lib/tag/tag.js

```diff
@@ -28,7 +28,9 @@ function Tag(impl, conf, innerHTML) {
 
   // grab attributes
   each(root.attributes, function(el) {
-    attr[el.name] = el.value
+    var val = el.value
+    // remember attributes with expressions only
+    if (brackets(/\{.*\}/).test(val)) attr[el.name] = val
```

Comment:
@rsbondi, `brackets()` returns an altered regex if the input is an regex if I'm not wrong. Also, the idea is to find at least 1 expression, so I think this regex should be fine.